hyperparams/ppo.yml

```diff
@@ -53,6 +70,24 @@ MountainCar-v0:
   n_epochs: 4
   ent_coef: 0.0
 
+# Tuned
+seals/MountainCar-v0:
+  normalize: true
+  n_envs: 16
+  n_timesteps: !!float 1e6
+  policy: 'MlpPolicy'
+  batch_size: 512
+  clip_range: 0.2
+  ent_coef: 6.4940755116195606e-06
+  gae_lambda: 0.98
+  gamma: 0.99
+  learning_rate: 0.0004476103728105138
+  max_grad_norm: 1
+  n_epochs: 20
+  n_steps: 256
```

Comment:
Interesting that this is a lot larger than for `MountainCar-v0` (although that one was perhaps not tuned)